metrics/thresholds_test.go

```diff
@@ -334,6 +337,219 @@ func TestThresholdsParse(t *testing.T) {
 	})
 }
 
+func TestThresholdsValidate(t *testing.T) {
+	t.Parallel()
+
+	t.Run("validating thresholds applied to a non-existing metric fails", func(t *testing.T) {
```

Comment:
```suggestion
	t.Run("FailNotExistingMetric", func(t *testing.T) {
```
All the other words seem redundant to me, IMHO

Also, this should generate something like the following line that doesn't seem consistent 
`TestThresholdsValidate/validating_thresholds_applied_to_a_non-existing_metric_fails`

Reply:
I removed the dash that created the inconsistency, but I prefer to keep the name as explicit as possible (especially for my future self) 👍🏻